phpunit.xml

```diff
@@ -11,13 +11,18 @@
          stopOnFailure               = "false">
 
     <testsuites>
-        <testsuite name="Alma PHP Client Test Suite">
-            <directory>tests</directory>
+        <testsuite name="Alma PHP Client Unit Test Suite">
+            <directory>tests/unit</directory>
+        </testsuite>
+    </testsuites>
+    <testsuites>
+        <testsuite name="Alma PHP Client Integration Test Suite">
+            <directory>tests/integration</directory>
         </testsuite>
     </testsuites>
     <php>
-        <env name="ALMA_API_KEY" value="sk_test_4C2ueBnMVMmgAwsS0y2cCE58"/>
-        <env name="ALMA_API_ROOT" value="http://localhost:1337"/>
+        <env name="ALMA_API_KEY" value="sk_test_1DNDPaxcvjDhR0A3063Xl75d"/>
```

Comment:
not fan to have API key in code :-/

Reply:
me neither, what about removing this file from the repo (.gitignore) and having a `phpunit.dist.xml` that we can copy ?

Reply:
:+1: